### PR TITLE
Cluster Updates Notifications

### PR DESCRIPTION
**Pull request by resmaa**
New sub page added to clusters.
 - Cluster Updates

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/320#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/320#" title="Author self-approved">resmaa</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: packages/docs/content/04-clusters/3-cluster-updates.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import Tabs from '@librarium/shared/src/components/ui/Tabs';
+import WarningBox from '@librarium/shared/src/components/WarningBox';
+import InfoBox from '@librarium/shared/src/components/InfoBox';
+import PointsOfInterest from '@librarium/shared/src/components/common/PointOfInterest';
+
+
+
+ 
+# Cluster Updates
+Spectro Cloud ensures the complete life cycle utilization  of its cluster for the users  by promoting strong day 2 operations. In this context cluster updates and integrations are of at most importance and could be done very easily. Cluster updates are all about updating the cluster profile features like packs, manifests, pack values, versions etc after the cluster is created. Each of these updates creates notifications to the user to check, confirm and incorporate the updates to the running cluster. A detailed overview of these updates and notifications pertaining to individual updates is enumerated in this session.
```
Comment (zulfilee) [inline packages/docs/content/04-clusters/3-cluster-updates.md:19]: for the user to check

File: packages/docs/content/04-clusters/3-cluster-updates.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+ 
+## Steps to follow:
+* Go to the Cluster profiles.
+* Select the cluster profile to be updated.
+* Make the pack updates pack updates and save.
```
Comment (zulfilee) [inline packages/docs/content/04-clusters/3-cluster-updates.md:24]: Can this be more clear?

File: packages/docs/content/04-clusters/3-cluster-updates.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+* A green button tagged ‘Updates Available’ can be seen on the right top of the screen, click open it.
+* A new window tagged ‘Available updates’ opens up.
+* This screen has information regarding all the profile updates done to that cluster.This window carries two types of information :
+ *  Profile Updates and notifications.
+ *  Current cluster values and incoming profile changes.
+* Users can verify the changes and press ‘confirm updates’ to complete the process of cluster updation.
+ 
+# Cluster Updates and Notifications
+
+Any updates to the Cluster Profile packs generates update notification on the Clusters. Users are required to verify the updates, resolve the pack values if any conflicts and confirm only if the updates are valid. Only on confirmation the updates are applied to the running Spectro Cloud clusters.
```
Comment (zulfilee) [inline packages/docs/content/04-clusters/3-cluster-updates.md:36]: if there are any conflicts?

File: packages/docs/content/04-clusters/3-cluster-updates.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+ 
+**Note:**
+Once a change is made at the cluster profile and notification update is not applied to the cluster (pending notification), then if the user decides to revert the changes at cluster profile then the pending notification on the cluster gets cleared automatically.
+
+Eg: new pack added - update notification generated  - keep the notification pending - remove the added pack - No updates visible on the cluster.
+ 
+# Cluster Profile Notification
+ 
+The registry updates such as new pack or helm chart release, existing pack values update generates the notification on the cluster profiles. If the user selects the Kubernetes pack with smart tag (eg: version 1.20.x) while adding the pack, any new Kubernetes release available at the registry auto updates the cluster profiles to update to the latest Kubernetes. Instead if the user selects a specific version (eg: version 1.20.2) of the pack then users need to update to the new version by updating the cluster profile manually. Any update to the cluster profile triggers the notification on the clusters and users should confirm the updates in order to apply the changes to the running kubernetes cluster.
```
Comment (zulfilee) [inline packages/docs/content/04-clusters/3-cluster-updates.md:66]: kubernetes or Kubernetes. We can have a standard convention. Kubernetes everywhere would be good

File: packages/docs/content/04-clusters/3-cluster-updates.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+ 
+|Update Notification     |Description|Example                   |
+|:---------------|:---------|:-----------------------|
+New Spectro Pack or Helm Chart Release |TA new pack or helm chart version is available at registry|"EventMessageRegistryPackNewVersion": "{{.Kubernetes}} {{.1.20.x}} is associated to version {{.1.20.5}}",|
+|Spectro Pack Values Update |Pack values modifications are  detected and notified to the cluster profiles.|"EventMessageRegistryPackUpdate": "{{.Kubernetes}} {{.1.20.5}} is updated : {{.packRevMessage}}",|
+|Pack Tag Delete|Removal of a pack from the registry.|"EventMessageRegistryPackTagDelete": {{.Kubernetes}} {{1.16.0 }} tag is delete|
+ 
+**Note:**
+
+*  Info Notification : If user doesn't customize the pack values at profile then system auto updates the cluster profile and show review notification to describe about the updates
```
Comment (zulfilee) [inline packages/docs/content/04-clusters/3-cluster-updates.md:76]: shows

File: packages/docs/content/04-clusters/3-cluster-updates.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+*  Update Notification: If user customize the pack values at profile then user must resolve the pack values and confirm the updates manually
```
Comment (zulfilee) [inline packages/docs/content/04-clusters/3-cluster-updates.md:77]: customizes

**Review by zulfilee — CHANGES_REQUESTED** [review]
Kindly take a look at the changes and address them

File: packages/docs/content/04-clusters/3-cluster-updates.md (hunk 1 above)
Comment (resmaa) [reply packages/docs/content/04-clusters/3-cluster-updates.md:19]: updated

File: packages/docs/content/04-clusters/3-cluster-updates.md (hunk 6 above)
Comment (resmaa) [reply packages/docs/content/04-clusters/3-cluster-updates.md:77]: all comments incorporated

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/320#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/320#pullrequestreview-625000514" title="Approved">zulfilee</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**